src/commands/build.js

```diff
@@ -0,0 +1,72 @@
+const Command = require('../base');
+const { flags: flagUtils } = require('@oclif/command');
+
+class TriggerBuildCommand extends Command {
+
+  /**
+   * Runs the build command to build the specific version on warehouse without promoting the version
+   *
+   * @async
+   * @returns {Promise} The result or error
+   */
+  async run() {
+    const { flags, args } = this.parse(TriggerBuildCommand);
+    const { env } = args; // package is reserved so we don't try to destructure it
+    const { pkg, version } = this.parsePackage(args.package);
+    const { wrhsHost, auth } = this.mergeConfig(flags);
+
+    if (!wrhsHost) {
+      this.error(this.missingHostError());
+    }
+    if (!version) {
+      this.error(this.missingVersionError());
+    }
+
+    const wrhs = this.wrhs(auth, { wrhsHost });
+
+    // Get build for environment for a given package name
+    return new Promise((resolve, reject) => {
+      wrhs.builds.trigger({ env, pkg, version, promote: flags.promote }, (err, response) => {
+        if (err) {
+          this.renderError('build', args.package, err);
+          return reject(err);
+        }
+        if (flags.json) {
+          this.log(JSON.stringify(response));
+          return resolve(JSON.stringify(response));
+        }
+
+        this.log('Build triggered successfully. See `get:status` command to get the status of a build.');
```

Comment:
Since we have the information, we should build the full `wrhs get:status` command for easy copying.